### ISSUE TITLE
Support :nextjournal.clerk/width on viewers rendered within Tap Inspector

### ISSUE THREAD

**Issue by ash-mcc** [nextjournal/clerk#430]
### Support :nextjournal.clerk/width on viewers rendered within Tap Inspector

**Background**

Ash: 

> I’ve some wide data that I’d like to tap> to the Tap Inspector
> but setting {::clerk/width :full} on the data’s viewers ( clerk/table , v/vl, etc.)
> seems to be ignored.
> Am I doing  it wrong, is it a bug, or does Tap Inspector not support :nextjournal.clerk/width? :thinking_face:

Martin: 

> doesn’t support it, yet. Issue welcome

**Comment by mk** [comment]
Hi @ash-mcc, please give `main` a try.

**Comment by ash-mcc** [comment]
Thanks Martin but with:

`(tap> (clerk/table {::clerk/width :full} [{:x 1}]))`

and the deps.edn:

```
{:paths ["src"]
 :deps  {org.clojure/clojure         {:mvn/version "1.11.1"} 
         
         ;; use the "Improve Tap Inspector (#447)" commit of the 12th April 2023
         io.github.nextjournal/clerk {:git/url "https://github.com/nextjournal/clerk.git"
                                      :sha     "c4f5fe11f7b4f1141654c7082f04d737f7192f1d"}
         }
 }
```

I see:
<img width="1186" alt="clerk-tap-width" src="https://user-images.githubusercontent.com/9269146/231874119-b5bf8bb0-4675-41b2-9d8f-e7fee6857f7a.png">

I was expecting that the table would be stretched to the full width of the browser window(?).

Cheers, Ash

**Comment by zampino** [comment]
> I was expecting that the table would be stretched to the full width 

@ash-mcc thanks. Something went missing in the latest commits of the PR, I'm having a look again.

**Comment by mk** [comment]
@ash-mcc I broke it with a final cleanup commit and @zampino fixed it with 97411f6afce9341aab46b560a79f695d3ad80a71, please give it another try.

**Comment by zampino** [comment]
@ash-mcc sorry, it should™️ now be fixed with 97411f6afce9341aab46b560a79f695d3ad80a71.

**Comment by ash-mcc** [comment]
Hi Martin and Andrea,  

...yes, that one works :-)

Thanks, Ash
